src/index.html

```diff
@@ -22,6 +22,349 @@
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#section-1"
+                class="nav__link is-active"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-2"
+                class="nav__link"
+              >
+                SAMSUNG
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-3"
+                class="nav__link"
+              >
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-4"
+                data-qa="nav-hover"
+                class="nav__link laptops"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-5"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-6"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-7"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#section-8"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="products">
+      <section
+        class="products__card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="IMAC image"
+          class="products__card_image"
+        />
+        <h2 class="products__card_name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="products__card_product-code">Product code: 195434</h3>
+        <div class="products__card_review-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="products__card_review-count">Reviews: 5</div>
+        </div>
+
+        <p class="products__card_price">
+          <span class="products__card_price-text">Price</span>
+          <span class="products__card_price-value">$2,199</span>
+        </p>
+        <a
+          class="products__card_buy-link"
+          href="#"
+          data-qa="card-hover"
+        >
```

Comment:
The 'Buy' link should have a descriptive `aria-label` for better accessibility, such as `aria-label="Buy APPLE A1419 iMac"`.